static/app/views/performance/trends/changeExplorerUtils/metricsTable.tsx

```diff
@@ -0,0 +1,413 @@
+    '',
+    0,
+    true,
+    beforeBreakpointErrors,
+    afterBreakpointErrors
+  );
+
+  const columnOrder = MetricColumnOrder.map(column => COLUMNS[column]);
+
+  return (
+    <GridEditable
+      data={[throughput, p50Events, p95Events, errors]}
+      columnOrder={columnOrder}
+      columnSortBy={[]}
+      grid={{
+        renderHeadCell,
+        renderBodyCell,
+      }}
+      location={location}
+      isLoading={
+        isLoadingBefore ||
+        isLoadingAfter ||
+        isLoading ||
+        isLoadingBeforeErrors ||
+        isLoadingAfterErrors
+      }
+    />
+  );
+}
+
+function getEventsRowData(
+  field: string,
+  rowTitle: string,
+  suffix: string,
+  nullValue: string | number,
+  wholeNumbers: boolean,
+  beforeData?: TableData,
+  afterData?: TableData
+): TableDataRow {
+  if (beforeData?.data[0][field] && afterData?.data[0][field]) {
+    return {
+      metric: rowTitle,
+      before: !wholeNumbers
+        ? toFormattedNumber(beforeData.data[0][field], 1) + ' ' + suffix
+        : beforeData.data[0][field],
+      after: !wholeNumbers
+        ? toFormattedNumber(afterData.data[0][field], 1) + ' ' + suffix
+        : afterData.data[0][field],
+      change: formatPercentage(
+        percentChange(
+          beforeData.data[0][field] as number,
+          afterData.data[0][field] as number
+        ),
+        1
+      ),
+    };
+  }
+  return {
+    metric: rowTitle,
+    before: nullValue,
+    after: nullValue,
+    change: '-',
+  };
+}
+
+function getTrendsRowData(
+  aggregateData: TrendsTransaction | undefined,
+  metric: TrendFunctionField
+): TableDataRow | undefined {
+  if (aggregateData) {
+    return {
+      metric: metric.toString().toUpperCase(),
+      before: aggregateData?.aggregate_range_1.toFixed(1) + ' ms',
+      after: aggregateData?.aggregate_range_2.toFixed(1) + ' ms',
+      change:
+        aggregateData?.trend_percentage !== 1
+          ? formatPercentage(aggregateData?.trend_percentage! - 1, 1)
+          : '-',
+    };
+  }
+  return undefined;
+}
+
+function getEventViewWithFields(
+  _organization: Organization,
+  eventView: EventView,
+  start: string,
+  end: string,
+  fields: AggregationKeyWithAlias[],
+  eventType: string,
+  transactionName: string,
+  dataset: DiscoverDatasets
+): EventView {
+  const newEventView = eventView.clone();
+  newEventView.start = start;
+  newEventView.end = end;
+  newEventView.statsPeriod = undefined;
+  newEventView.dataset = dataset;
+  newEventView.query = 'event.type:' + eventType + ' transaction:' + transactionName;
+  newEventView.additionalConditions = new MutableSearch('');
+
+  const chartFields: QueryFieldValue[] = fields.map(field => {
+    return {
+      kind: 'function',
+      function: [field, '', undefined, undefined],
+    };
+  });
+
+  return newEventView.withColumns(chartFields);
+}
+
+function toFormattedNumber(numberString: ReactText, decimal: number) {
+  return (numberString as number).toFixed(decimal);
```

Comment:
I noticed this takes a `ReactText` and asserts it as a number when calling `toFixed()`, can we just make the type of `numberString` a `string` because that seems like it agrees with the types?

This is kind of a roundabout way to handle it but you can do this to apply `toFixed`:
```
return parseInt(numberString, 10).toFixed(decimal);
```